sig-instrumentation/annual-report-2022.md

```diff
@@ -4,118 +4,167 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-   -
-   -
-   -
+   - Tracing support for the Kubernetes api-server and kubelet have graduated to beta.
+   - We've introduced a KEP for Kubernetes control-plane component SLIs.
+   - The metrics stability framework is being extended and now support `BETA` stability levels and auto-generated documentation.
+   - Contextual logging has graduated to beta.
 
 2. What initiatives are you working on that aren't being tracked in KEPs?
 
-   -
-   -
-   -
-
-
-
-3. KEP work in 2022 (v1.24, v1.25, v1.26):
-  - alpha:
-    - [2831 - Kubelet OpenTelemetry Tracing](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2831-kubelet-tracing) - v1.25
-    - [3077 - Contextual logging](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/3077-contextual-logging) - v1.24
-    - [3466 - Kubernetes Component Health SLIs](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/3466-kubernetes-component-health-slis) - v1.26
-  - stable:
-    - [2845 - Deprecate klog specific flags in Kubernetes components](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components) - v1.26
-
+   - Bi-weekly triage meeting
+   - Subprojects (see below)
+
+3. KEP work in 2022 (1.x, 1.y, 1.z):
+
+<!--
+In future, this will be generated from kubernetes/enhancements kep.yaml files
+1. with SIG as owning-sig or in participating-sigs
+2. listing 1.x, 1.y, or 1.z in milestones or in latest-milestone
+-->
+   - Stable
+     - 1.25 External metrics gradation 
+     - 1.25 Custom metrics graduation
```

Comment:
This didn't happen, the APIs are still in Beta atm and I didn't get any bandwidth to graduate them all